binding/python/examples/theano/logistic_regression.py

```diff
@@ -318,6 +331,13 @@ def sgd_optimization_mnist(learning_rate=0.13, n_epochs=1000,
     ######################
     print('... building the model')
 
+    mv.init()
+    WORKER_ID = mv.worker_id()
```

Comment:
We should be consistent with using `WORKER_ID` or `worker_id`, or we should use `WORKER_ID` when it is used in module level (global variable) and `worker_id` for local variable.